SpatialGDK/Source/SpatialGDK/Private/SpatialView/SubView.cpp

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialView/SubView.h"
+
+#include "Utils/ComponentFactory.h"
+
+namespace SpatialGDK
+{
+FSubView::FSubView(const Worker_ComponentId& TagComponentId, FFilterPredicate Filter, const EntityView* View, FDispatcher& Dispatcher,
```

Comment:
Also nit: visilibity can be an issue - please append `In` to input variable names that have otherwise identical names to member variables

Reply:
does in go at the front or the back

Reply:
Front

Reply:
ins are in